### PR TITLE
Endpoint management RBAC

### LINKED ISSUE #2965: Endpoint management RBAC
## Description 

Related:
* https://github.com/elastic/security-team/issues/5446
* https://github.com/elastic/kibana/pull/149446

_from https://github.com/elastic/security-team/issues/5446_:

> Currently, all endpoint management functions require superuser role. Administrators and operations managers should be able to give finer grained access to those on their team, so that they do not have to give all-or-nothing permissions.
> 
> Endpoint management functions include:
> 
> Endpoint List (All | Read | None)
> Trusted Applications (All | Read | None)
> Host Isolation Exceptions (All | Read | None)
> Blocklist (All | Read | None)
> Event Filters (All | Read | None)
> Elastic Defend Policy Management (All | Read | None)
> Response Actions History (All | Read | None)

_image from https://github.com/elastic/kibana/pull/149446:_ 
image

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2965.

Previews:
- [Elastic Defend feature privileges](https://security-docs_3040.docs-preview.app.elstc.co/guide/en/security/master/endpoint-management-req.html) - This page has the substantial updates for this feature (added all the new privileges, revised general instructions)
- Very small changes in lots of other pages -- see the diff on Files changed tab (replaced `superuser` with feature-specific privs in prereq sidebars, other references)

### CODE REVIEW

File: docs/getting-started/defend-feature-privs.asciidoc [hunk 1]
```diff
@@ -7,22 +7,44 @@ Roles and privileges are configured in *Stack Management* -> *Roles* in {kib}. F
...
 
-To grant access, set the privileges for the following sub-features under the *Security* feature in the *{kib} privileges* configuration UI:
-
 [role="screenshot"]
 image::images/endpoint-privileges.png[Configuring privileges in Kibana,75%]
 
+To grant access, select *All* for the *Security* feature in the *{kib} privileges* configuration UI, then turn on the *Customize sub-feature privileges* switch. For each of the following sub-feature privileges, select the type of access you want to allow:
+
+* `All`: Users have full access to the feature and can perform all available actions and configuration.
+* `Read`: Users can view the feature, but they can't perform any actions or configuration. (Not all privileges have this option.)
```
Comment (nastasha-solomon) [inline docs/getting-started/defend-feature-privs.asciidoc:16]:
```suggestion
* `Read`: Users can view the feature, but they can't perform any actions or configuration. (Some sub-features don't have this privilege option.)
```

File: docs/getting-started/defend-feature-privs.asciidoc [hunk 2]
```diff
@@ -7,22 +7,44 @@ Roles and privileges are configured in *Stack Management* -> *Roles* in {kib}. F
...
+
 [cols="1,1",width="100%"]
 |==============================================
-| *Response Actions History* 
-a| Allow users to access the <<response-actions-history,response actions history>> for endpoints.
+| *Endpoint List*
+| Access the <<admin-page-ov,Endpoints>> page, listing all hosts running {elastic-defend} and their relevant integration details.
+
+| *Trusted Applications*
+| Access the <<trusted-apps-ov,Trusted Applications>> page to mitigate conflicts with other software, such as other antivirus or endpoint security applications.
```
Comment (benironside) [inline docs/getting-started/defend-feature-privs.asciidoc:25]:
```suggestion
| Access the <<trusted-apps-ov,Trusted Applications>> page to remediate conflicts with other software, such as other antivirus or endpoint security applications.
```

**Review by benironside — COMMENTED** [review]
Iconic PR, very cool, this content needs to exist. Left some minor suggestions for your consideration

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

File: docs/getting-started/defend-feature-privs.asciidoc [hunk 3]
```diff
@@ -7,22 +7,51 @@ Roles and privileges are configured in *Stack Management* -> *Roles* in {kib}. F
...
 NOTE: {elastic-defend}'s feature privileges must be assigned to *All Spaces*. You can't assign them to an individual {kib} space. 
 
-To grant access, set the privileges for the following sub-features under the *Security* feature in the *{kib} privileges* configuration UI:
-
 [role="screenshot"]
 image::images/endpoint-privileges.png[Configuring privileges in Kibana,75%]
 
+To grant access, select *All* for the *Security* feature in the *{kib} privileges* configuration UI, then turn on the *Customize sub-feature privileges* switch. For each of the following sub-feature privileges, select the type of access you want to allow:
+
+* *All*: Users have full access to the feature and can perform all available actions and configuration.
```
Comment (jmikell821) [inline docs/getting-started/defend-feature-privs.asciidoc:15]: `configuration` sounds like a verb here, so adding one before. 
```suggestion
* *All*: Users have full access to the feature, which includes performing all available actions and managing configuration.
```

File: docs/getting-started/defend-feature-privs.asciidoc [hunk 4]
```diff
@@ -7,22 +7,51 @@ Roles and privileges are configured in *Stack Management* -> *Roles* in {kib}. F
...
+
 [cols="1,1",width="100%"]
 |==============================================
-| *Response Actions History* 
-a| Allow users to access the <<response-actions-history,response actions history>> for endpoints.
+| *Endpoint List*
+| Access the <<admin-page-ov,Endpoints>> page, which lists all hosts running {elastic-defend}, and associated integration details.
+
+| *Trusted Applications*
+| Access the <<trusted-apps-ov,Trusted Applications>> page to remediate conflicts with other software, such as other antivirus or endpoint security applications.
```
Comment (jmikell821) [inline docs/getting-started/defend-feature-privs.asciidoc:25]:
```suggestion
| Access the <<trusted-apps-ov,Trusted Applications>> page to remediate conflicts with other software, such as antivirus or endpoint security applications.
```

**Review by jmikell821 — COMMENTED** [review]
Looks great! Just a few small comments, thanks!

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by joepeeples** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by caitlinbetz — APPROVED** [review]

**Review by kevinlog — APPROVED** [review]

**Pull request merged**